composer/loggers/mlflow_logger.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Log to `MLFlow <https://www.mlflow.org/docs/latest/index.html>."""
+
+from __future__ import annotations
+
+import pathlib
+from typing import Any, Dict, Optional, Union
+
+from composer.core.state import State
+from composer.loggers.logger import Logger
+from composer.loggers.logger_destination import LoggerDestination
+from composer.utils import MissingConditionalImportError, dist
+
+__all__ = ['MLFlowLogger']
+
+
+class MLFlowLogger(LoggerDestination):
+    """Log to `MLFlow <https://www.mlflow.org/docs/latest/index.html>`_.
+
+    Args:
+        experiment_name: (str, optional): MLFLow experiment name,
+        run_name: (str, optional): MLFlow run name.
+        tracking_uri (str | pathlib.Path, optional): MLFlow tracking uri, the URI to the
+            remote or local endpoint where logs are stored (If none it is set to `./mlruns`)
+        rank_zero_only (bool, optional): Whether to log only on the rank-zero process
+            (default: ``False``).
```

Comment:
This is actually set to default to True (and actually the comet logger docstring is wrong also).

Reply:
fixed